invokeai/app/services/image_record_storage.py

```diff
@@ -238,6 +251,17 @@ def update(
                     """,
                     (changes.session_id, image_name),
                 )
+
+            # Change the image's `is_intermediate`` flag
+            if changes.session_id is not None:
```

Comment:
Should this be `changes.is_intermediate`?

Reply:
yes it should be